src/test/java/htsjdk/samtools/util/HistogramTest.java

```diff
@@ -5,17 +5,18 @@
 import org.testng.annotations.Test;
 
 import java.io.IOException;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.HashSet;
 
 import static java.lang.Math.abs;
+import static java.lang.StrictMath.pow;
 
-/**
- *
- */
 public class HistogramTest {
 
```

Comment:
The java serialization issue I was seeing was due to a Histogram with a non-serializable anonymous comparator; it is only kryo serialization that fails always, so I don't have a case that fails without a kryo dependency. FWIW I did test this branch against my gatk branch that uses a custom kryo serializer to work around the kryo issues, and with these changes kryo is able to serialize Histograms without the custom serializer.